packages/wrangler/src/is-interactive.ts

```diff
@@ -6,7 +6,7 @@ import { CI } from "./is-ci";
  * or you're piping values from / to another process, etc
  */
 export default function isInteractive(): boolean {
```

Comment:
many CLIs I've used respect CI=1 unless FORCE_COLOR=1 - I'm surprised we don't do that in wrangler

Reply:
The library that wrangler uses `is-ci` doesnt' seem to respect CI=1 unfortunately.

Reply:
We actually do, it just uses this external is-ci library (which imports the ci-info library): https://github.com/watson/ci-info/blob/master/index.js#L60